eli5/sklearn/permutation_importance.py

```diff
@@ -214,8 +214,13 @@ def _cv_scores_importances(self, X, y, groups=None, **fit_params):
         cv = check_cv(self.cv, y, is_classifier(self.estimator))
         feature_importances = []  # type: List
         base_scores = []  # type: List[float]
+        weights = fit_params.get('sample_weight', None)
+        fit_params.pop('sample_weight', None)
```

Comment:
An minor improvement is possible here: ``pop`` already returns what it popped, so these two lines could be reduced to one:
```
 weights = fit_params.pop('sample_weight', None)
```

Reply:
Done